docs/GettingStarted.md

```diff
@@ -36,7 +36,7 @@ To contribute, check out the [contribution guidelines][contribution docs] and [d
      - [Concurrent Ruby](#concurrent-ruby)
      - [Dalli](#dalli)
      - [DelayedJob](#delayedjob)
-     - [Elastic Search](#elastic-search)
+     - [Elasticsearch](#elastic-search)
```

Comment:
I think you need to update `#elastic-search` to `#elasticsearch` too, because the anchor link will be broken to the section you updated. (Dashes indicate spaces.)